velox/connectors/hive/HiveDataSink.h

```diff
@@ -99,6 +99,7 @@ class HiveWriterParameters {
   enum class UpdateMode {
     kNew, // Write files to a new directory.
     kOverwrite, // Overwrite an existing directory.
+    kAppend, // Append to an existing directory.
```

Comment:
It looks like this is supported only for unpartitioned tables. Should we add a comment about that?

Also, it seems like there is no 'append' logic other then passing this flag back to coordinator.

Reply:
I have added a comment.